drivers/espi/host_subs_npcx.c

```diff
@@ -437,6 +498,58 @@ static void host_port80_init(void)
 }
 #endif
 
+#if defined(CONFIG_ESPI_PERIPHERAL_CUSTOM_OPCODE)
+static void host_cus_opcode_enable_interrupts(void)
+{
+	/* Enable host KBC sub-device interrupt */
+#if defined(CONFIG_ESPI_PERIPHERAL_8042_KBC)
```

Comment:
could IS_ENABLED be used to clean up all of these here?

Reply:
Sure. Please refer [PR30919](https://github.com/zephyrproject-rtos/zephyr/pull/30919)